src/DocumentFormat.OpenXml/Packaging/OpenXmlPart.cs

```diff
@@ -78,13 +78,16 @@ internal void Load(OpenXmlPackage? openXmlPackage, OpenXmlPart? parent, Uri uriT
 
             if (IsContentTypeFixed && metroPart.ContentType != ContentType)
             {
-                var errorMessage = SR.Format(
+                if (!((metroPart.ContentType == "model/gltf.binary") && (ContentType == "model/gltf-binary")))
```

Comment:
can you not special case this? Maybe add a virtual method to OpenXmlPart that `IsValidContentType` that you can override in the type that allows this contenttype to also check that

Reply:
@twsouthwick  thanks for the suggestion, I added the virtual method for this and fixed it in the Model3DReferenceRelationshipPart class. please review.